diff --git a/feeds/morse/essentials/netifd-morse/lib/wifi/morse.sh b/feeds/morse/essentials/netifd-morse/lib/wifi/morse.sh
--- a/feeds/morse/essentials/netifd-morse/lib/wifi/morse.sh
+++ b/feeds/morse/essentials/netifd-morse/lib/wifi/morse.sh
@@ -116,6 +116,11 @@ EOF
 				morse,halowlink2)
 					bcf=bcf_mf15457.bin
 				;;
+				asiarf,ap7622-wh1|\
+				asiarf,ap7621-004-v3|\
+				asiarf,mm610x-h06-spi-and-sdio)
+					bcf=arfhl_d1164_bcf_mf10220_au_v3.bin
+				;;
 				*)
 					if [[ $path  = *usb* ]]; then
 						bcf=bcf_mf15457.bin
-- 
2.34.1

